weex_core/Source/core/bridge/script/core_side_in_script.cpp

```diff
@@ -71,18 +71,11 @@ std::unique_ptr<ValueWithType> CoreSideInScript::CallNativeModule(
     const char *page_id, const char *module, const char *method,
     const char *arguments, int arguments_length, const char *options,
     int options_length) {
-  std::unique_ptr<ValueWithType> ret(new ValueWithType);
-  ret->type = ParamsType::INT32;
-  ret->value.int32Value = -1;
+  std::unique_ptr<ValueWithType> ret(new ValueWithType((int32_t)-1));
   if (page_id != nullptr && module != nullptr && method != nullptr) {
-    RenderManager::GetInstance()->CallNativeModule(page_id, module, method,
-                                                   arguments, arguments_length,
-                                                   options, options_length);
-    return WeexCoreManager::Instance()
```

Comment:
Add

    return WeexCoreManager::Instance()
        ->getPlatformBridge()
        ->platform_side()
        ->CallNativeModule(page_id, module, method, arguments, arguments_length,
                           options, options_length);

back

Reply:
I'm agree with you , but 
WeexCoreManager::Instance()
    ->getPlatformBridge()
    ->platform_side()
    ->CallNativeModule(page_id, module, method, arguments, arguments_length,
                       options, options_length);

has been added to `RenderManager::GetInstance()->CallNativeModule`, and page CallNativeModule may do some custom action. So it ok